contoso-bicycle/README.md

```diff
@@ -0,0 +1,115 @@
+
+# Case study – Contoso Bicycle
+
+Contoso Bicycle is a fictitious company. The company is a small and fast-growing startup that provides online web services to its clientele in the west coast, North America. The web services were deployed to the cloud from the get-go. They have no on-premises data centers or legacy applications. Here's the brief cloud profile:
+
+- Have several workloads running and operating in Azure.
+- Use Azure Active Directory for identity management.
+- Knowledgeable about containers and have considered them for application development.
+- Aware of Kubernetes as a well-known container orchestration.
+- Researched AKS as a possibility.
+
+The IT teams need guidance about architectural recommendations for running their web services in an AKS cluster.
+
+Based on the company's profile and business requirements, we've created a [reference architecture](https://aka.ms/architecture/aks-baseline) that serves as a baseline with focus on security. The architecture is accompanied with the implementation found in this repo. We recommend that you start with this implementation and add components based on your needs.
+
+## Organization structure
+
+Contoso Bicycle has a single IT Team with these sub teams.
+
+![Contoso teams](contoso-teams.svg)
+
+### Architecture team
+
+Work with the line of business from idea through deployment into production. They understand all aspects of the Azure components: function, integration, controls, and monitoring capabilities. The team evaluates those aspects for functional, security, and compliance requirements. They coordinate and have representation from other teams. Their workflow aligns with Contoso's SDL process.
+
+### Development team
+
+Responsible for developing Contoso’s web services. They rely on the guidance from the architecture team about implementing cloud design patterns. They own and run the integration and deployment pipeline for the web services.
+
+### Security team
+
+Review Azure services and workloads from the lens of security. Incorporate Azure service best practices in configurations. They review choices for authentication, authorization, network connectivity, encryption, and key management and, or rotation. Also, they have monitoring requirements for any proposed service.
+
+### Identity team
+
+Responsible for identity and access management for the Azure environment. They work with the Security and Architecture teams for use of Azure Active Directory, role-based access controls, and segmentation. Also, monitoring service principles for service access and application level access.
+
+### Networking team
+
+Make sure that different architectural components can talk to each other in a secure manner. They manage the hub and spoke network topologies and IP space allocation.
+
+### Operations team
+
+Responsible for the infrastructure deployment and day-to-day operations of the Azure environment.
+
+## Business requirements

```

Comment:
aren't we already covering this whole topic from the docs? Could we just cross-link or we do really want this in here.

Reply:
Not as sucinctly -- this list is scattered to the wind now.